src/main/java/xades4j/verification/XadesVerifierImpl.java

```diff
@@ -183,9 +183,20 @@ public XAdESVerificationResult verify(Element signatureElem, SignatureSpecificVe
 
         /* Certification path */
 
-        Date validationDate = getValidationDate(qualifPropsData, signature, verificationOptions);
         CertRef signingCertRefAttempt = tryGetSigningCertificateRef(qualifPropsData);
         KeyInfoRes keyInfoRes = KeyInfoProcessor.process(signature.getKeyInfo(), signingCertRefAttempt, this.x500NameStyleProvider);
+
+        Date validationDate;
+        if(verificationOptions.checkValidity())
+        {
+            validationDate = getValidationDate(qualifPropsData, signature, verificationOptions);
+        }
+        else
+        {
+            // use not before of signature certificate instead of sig date or not
+            validationDate = keyInfoRes.certSelector.getCertificate().getNotBefore();

```

Comment:
Not sure about this, because this date in unrelated to the validation instant. At most this could use `verificationOptions.getDefaultVerificationDate()`, but the name for `checkValidity` is misleading. 

TBH, since you don't really need this, I think we should remove the option for verification to be on the safe side.

Reply:
Yeah, I thought the same. I will remove this check an change the test.